morpheushelper/cogs/rules.py

```diff
@@ -16,6 +16,18 @@ class RulesCog(Cog, name="Rule Commands"):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    @staticmethod
+    async def get_message_cancel(bot: Bot, channel: TextChannel, member: Member) -> tuple[Optional[str],
+                                                                                          list[Optional[File]]]:
```

Comment:
shouldn't this be a list of files instead of a list of optional files? ([PyDrocsid/util.py#L114](https://github.com/Defelo/PyDrocsid/blob/master/PyDrocsid/util.py#L114))
```suggestion
                                                                                          list[File]]:
```